db/seeds/protocols/student_control_group.rb

```diff
@@ -53,7 +53,7 @@
 db_measurement.save!
 
 # Add nameting/enquete
-nm_name = 'nameting studenten'
+nm_name = 'nameting studenten controle'
```

Comment:
ik dacht dat ze de zelfde nameting zouden krijgen.

Reply:
Nee daar staan vragen over de begeleiding in.